libcodechecker/server/database/database.py

```diff
@@ -59,7 +59,7 @@ def __enter__(self):
         except Exception as ex:
             LOG.debug("Connection error")
             LOG.debug(ex)
-            self.error = DBStatus.FAILED_TO_CONNECT
+            self.db_error = DBStatus.FAILED_TO_CONNECT
```

Comment:
`DBContext.error` is checked at multiple places, if we rename it we should rename it everywhere.

Reply:
Ok, thanks. Please initialize `self.db_error` in `__intit__`